main.go

```diff
@@ -296,7 +296,7 @@ func initFlags(fs *pflag.FlagSet) {
 
 	fs.IntVar(&restConfigBurst, "kube-api-burst", 30,
 		"Maximum number of queries that should be allowed in one burst from the controller client to the Kubernetes API server. Default 30")
-	flag.StringVar(&tlsOptions.TLSMinVersion, "tls-min-version", TLSVersion12,
+	fs.StringVar(&tlsOptions.TLSMinVersion, "tls-min-version", TLSVersion12,
```

Comment:
CAPM3 it is `flag` + `fs`, and in BMO/IPAM it is `flag` + `flag`, but similarly all the other flags are `fs` in IPAM/BMO. Those need to be fixed as well.

Some cherry-picks to relevant branches too?

Reply:
IPAM I am fixing it already here https://github.com/metal3-io/ip-address-manager/pull/398, for BMO I can check later. We can cherry-pick this.

Reply:
Nice!